nansat/utils.py

```diff
@@ -205,11 +205,10 @@ def add_logger(logName='', logLevel=None):
     # otherwise create and add a new handler
     if len(logger.handlers) == 0:
         # create console handler and set level to debug
-        ch = logging.StreamHandler()
+        ch = logging.StreamHandler(sys.stdout)
         # create formatter
-        formatter = logging.Formatter('%(asctime)s|%(levelno)s|%(module)s|'
-                                      '%(funcName)s|%(message)s',
-                                      datefmt='%I:%M:%S')
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(threadName)s - '
+                                      '%(levelname)s - %(message)s')
```

Comment:
You don't need the threadName field, nansat is single-threaded

Reply:
actually is there any need to change the format at all?

Reply:
In order to make it in a unified format when we start the process from harvesting, this helps to read the logs easily,all levels of nansat,harvesting and run_container.py MUST have same format of log. Because they are unified when call from each other. Reading logs with different format is difficult. so it is better to keep it. The thread is unknown, or sth like that,it does not matter, let it be.